app/models/manageiq/providers/redhat/inventory/collector.rb

```diff
@@ -157,4 +157,40 @@ def collect_dc_domains(dc)
       connection.follow_link(dc.storage_domains)
     end
   end
+
+  def collect_vm_disks(vm)
+    manager.with_provider_connection(VERSION_HASH) do |connection|
+      disk_attachments = connection.follow_link(vm.disk_attachments)
+      disk_attachments.collect do |disk_attachment|
+        connection.follow_link(disk_attachment.disk)
+      end
+    end
+  end
+
+  def vm_or_template_by_path(path)
+    uuid = ::File.basename(path, '.*')
+    vm      = vm_by_uuid(uuid)
```

Comment:
Lots of extra spaces here, did one of these variables used to have a longer name?